opensearch/roles_mapping.yml

```diff
@@ -0,0 +1,7 @@
+_meta:
+  type: rolesmapping
+  config_version: 2
+
+all_access:
+  backend_roles:
+  - opendistro_security_anonymous_backendrole
```

Comment:
This is magically created by the security setup?
We're going to get a username/password in a followup PR?

Reply:
Both 'all_access' and 'opendistro_security_anonymous_backendrole' seem to be supplied out of the box.  Here they're linked together to achieve the same behavior we had without the security plugin.  The current task is to communicate over SSL.  We can create another task for passwords once we figure out how we want to do user management and authentication.  OpenSearch seems to prefer using client certificates over username/password.